integreat_cms/api/v3/feedback/event_feedback.py

```diff
@@ -67,10 +68,11 @@ def event_feedback_internal(data, region, language, comment, rating, is_technica
     :return: JSON object according to APIv3 single page feedback endpoint definition
     :rtype: ~django.http.JsonResponse
     """
-    event_translation_slug = data.get("slug")
+    # Remove date from the slug for recurrung events
+    event_translation_slug = re.sub("\$\d{4}-\d{2}-\d{2}$", "", data.get("slug"))
```

Comment:
You should probably mark the string as a raw string literal:
```suggestion
    event_translation_slug = re.sub(r"\$\d{4}-\d{2}-\d{2}$", "", data.get("slug"))
```

Reply:
yea, already done.
it worked without it though